packages/worker/src/workflow/vm.ts

```diff
@@ -256,7 +255,27 @@ export class VMWorkflowCreator implements WorkflowCreator {
       });
     }
 
-    return new this(script, sourceMap, isolateExecutionTimeoutMs) as InstanceType<T>;
+    return new this(script, isolateExecutionTimeoutMs) as InstanceType<T>;
+  }
+
+  private static async makeScriptWithSourceMap(code: string) {
```

Comment:
Might as well make this protected in case anyone wants to extend this (granted we won't provide API safety for protected members)

Reply:
Could use some comments in this function